src/app/forms/components/dynamic-form-group/dynamic-form-group.component.scss

```diff
@@ -8,3 +8,9 @@
     margin-top: 30px;
   }
 }
+
+.subheading-container {
```

Comment:
Could this be done with gov-uk classes ? https://design-system.service.gov.uk/styles/layout/

Reply:
The gov-uk classes use set percentages such as full, one-half, one-third, two-thirds, one-quarter, three-quarters, etc.

Whereas, I want my 2 elements to have the minimum width possible and fill the middle with space, regardless of screen size.